### ISSUE TITLE
Memory  Leak!!!

### ISSUE THREAD

**Issue by 53RP3N7** [GiviMAD/rustpotter#10]
It looks like that the application has a memory leak, please reply if you need any additional information or not. At the moment I can confirm that the RAM usage after the application starts, increases in seconds, and gradually is using the all the available recourses.

**Comment by GiviMAD** [comment]
Yes please, I didn't see that happening while using the cli.
What kind of model are you using and in which platform?

**Comment by 53RP3N7** [comment]
Sorry, just saw the message, I'm on a windows machine, and I'm using the application build that you've posted in Telegram. Okay I just went back and relaunched it again, and it seems like to be working better I guess, the first week that I've tested, after the launch of the application my fans started going crazy after like 10 minutes, so I've started monitoring the RAM usage of the application, and it was gradually rising. But while writing this I wanted to  test the timing again, but it seems like it's not using that much RAM currently, like after several actions with the commands, it's using more and more RAM, however, still not getting to 300 MB, but previously it used to rise in the stand-by mode and got up to 800 MB before I turned it off just in case. If possible I'd like to keep this ticket open, and will continue to monitor the issue, in case it'll be an ongoing one, I'll keep you posted, and if the issue, is not replicated anymore I'll close the ticket :) Also thank you for this anyway and thanks for the quick response :)
